assets/app/views/settings.html

```diff
@@ -1,119 +1,236 @@
 <div class="content">
   <project-page>
-    <div>
-      <div class="page-header page-header-bleed-right">
-        <delete-link
-          class="pull-right fa-lg"
-          resource-type="project"
-          resource-name="{{project.metadata.name}}"
-          project-name="{{project.metadata.name}}"
-          display-name="{{(project | displayName)}}"
-          alerts="alerts"
-          button-only>
-        </delete-link>
-        <h1>Project Settings</h1>
-      </div>
+    <div class="breadcrumb"></div>
+    <alerts alerts="alerts"></alerts>
+    <div class="tile__">
```

Comment:
No, disabled/enable the 'tile'.  I was considering using a tile like we don on DCs and other things, but im not sure it makes sense for settings.  I will remove, unless anyone thinks we should keep it.

Screenshot attached for comparison

![screen shot 2016-01-15 at 9 54 52 am](https://cloud.githubusercontent.com/assets/280512/12355977/21e657f8-bb6e-11e5-9c45-dd28ccd631a8.png)